saleor/order/tests/test_order_actions_refund_products.py

```diff
@@ -218,7 +230,11 @@ def test_create_refund_fulfillment_custom_amount(
     for line in fulfillment_lines:
         assert line.quantity == original_quantity.get(line.pk) - 2
     mocked_refund.assert_called_once_with(
-        payment_dummy, ANY, amount=amount, channel_slug=fulfilled_order.channel.slug
+        payment_dummy,
+        ANY,
+        amount=amount,
+        channel_slug=fulfilled_order.channel.slug,
+        refund_data=ANY,
```

Comment:
Why any of these tests do not check what is provided in the `refund_data` parameter?

Reply:
Fixed in current commit.